plugin/trino-hive/src/test/java/io/trino/plugin/hive/metastore/glue/TestGlueConverter.java

```diff
@@ -403,11 +402,7 @@ void testDeltaTableNonNullStorageDescriptor()
                 .build();
         assertThat(table.storageDescriptor()).isNotNull();
         io.trino.metastore.Table trinoTable = GlueConverter.fromGlueTable(table, table.databaseName());
-        assertThat(trinoTable.getDataColumns().stream()
-                .map(Column::getName)
-                .collect(toImmutableSet())).isEqualTo(glueTable.storageDescriptor().columns().stream()
-                .map(software.amazon.awssdk.services.glue.model.Column::name)
-                .collect(toImmutableSet()));
+        assertThat(trinoTable.getDataColumns().size()).isEqualTo(1);
```

Comment:
AssertJ provides `hasSize` method.

Reply:
Done, I also fixed other occurrences in a separate commit.